internal/gcsx/random_reader_test.go

```diff
@@ -55,7 +58,7 @@ type checkingRandomReader struct {
 	wrapped *randomReader
 }
 
-func (rr *checkingRandomReader) ReadAt(p []byte, offset int64) (int, bool, error) {
```

Comment:
New tests are added in random_reader_stretcher_test.go. That file is missing here.